src/index.js

```diff
@@ -7,6 +7,10 @@ const supportMultiple = (typeof document !== 'undefined' && document && document
   'multiple' in document.createElement('input') :
   true;
 
+const walkDirectory = Symbol('walk directory');
```

Comment:
I see, but we want the same code style for this one. Can you update your PR please?